packages/components/organisms/f-status-banner/src/components/StatusBanner.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+    <div
+        :class="$style['c-statusBanner']"
+        data-test-id="statusBanner">
+        {{ copy.text }}
+    </div>
+</template>
+
+<script>
+import { globalisationServices } from '@justeat/f-services';
+import tenantConfigs from '../tenants';
+/* eslint-disable */
+// import StatusBannerServiceApi from '../services/StatusBannerServiceApi';
```

Comment:
Do you need this?

Reply:
I'll probably end up removing this in the next pr tbh. Just left it in incase I end up needing it for the next set of changes.